media/videostitcher/stitcher_test.go

```diff
@@ -229,6 +232,38 @@ func getUUID64() (string, error) {
 	return strings.ReplaceAll(uuid, "-", ""), nil
 }
 
+// Curl the playURI first. The last line of the response will contain a
+// renditions location. Return the renditions.
+func curlPlayURI(playURI string) ([]string, error) {
```

Comment:
I think you should just call this `getPlayURI`, you're not actually shelling out to curl but rather using net/http Get. Also, is there a particular reason to use a raw http client for this rather than a client library of some kind?

Reply:
Renamed the function.

Getting the playURI simulates a single media player retrieving the media playlist manifest. There is no client library for this type of request.